core/process.py

```diff
@@ -1002,13 +1066,21 @@ class Pipeline(Job):
   $(foo | bar)
   foo | bar | read v
```

Comment:
I just pulled and tested out

```
sleep | cat
sleep | echo hi
```

It seems like they both can't be suspended, and that behavior is OK with me

But we should document that somewhere ?  It doesn't seem obvious to me from this Pipeline class

Reply:
Ah yes this is intended. We should definitely document. I'll also spend some time later today seeing if we can pull off the "pipelines can't be suspended" log like zsh

Reply:
Agh I forgot that zsh does this by letting the children get stopped, printing the message when `wait()` returns with WIFSTOPPED, then sending SIGCONT to the children.

I'm not sure why exactly, but that feels awkward. Maybe it's less awkward than any convoluted group magic we'd have to do otherwise, since that stuff is hard enough to keep straight as it is... Maybe it's best to just document this